src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -76,10 +76,14 @@ Copyright (c) .NET Foundation. All rights reserved.
            targets. -->
       <Reference Remove="%(_NETStandardLibraryNETFrameworkLib.FileName)" />
 
-      <Reference Include="@(_NETStandardLibraryNETFrameworkLib)">
-        <!-- netfx.force.conflicts is only needed at compile time. -->
-        <Private Condition="'%(FileName)' == 'netfx.force.conflicts'">false</Private>
+      <Reference Include="@(_NETStandardLibraryNETFrameworkLib.FileName)">
+        <HintPath>%(_NETStandardLibraryNETFrameworkLib.Identity)</HintPath>
```

Comment:
Why use the HintPath?  Why not just `<Reference Include="full path"`?

Reply:
> This also uses the simple name as the identity for the `Reference` items, and uses the `HintPath` metadata for the full path.  This should resolve at least some cases of #1499.

Reply:
Ah, I see it in your description now - for #1499.

Reply:
Yeah, I wonder if we want NuGet to do the same thing 🤔   I can't recall why the NuGet task originally used RawFileName given that was inconsistent with what the project system was doing, @jasonmalinowski might know.